packages/core/src/lib/map/map.ts

```diff
@@ -330,18 +346,40 @@ Original error: ${originalError}`;
           nestedSourceMemberKey,
           nestedDestinationMemberKey
         );
+
+        // nested mutate
+        const destinationMemberValue = getMemberFn?.(destinationMemberPath);
```

Comment:
suggestions: optional chaining is great but for library code, optional chaining generates a lot of code. Can we do null check for `getMemberFn` manually?

Reply:
nvm. It's only bad when we have more than 1 optional chaining.